File: gluecodium/src/test/resources/smoke/internal_fields/output/android/com/example/smoke/PublicFieldsAllInit.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class PublicFieldsAllInit {
    @NonNull
    public String publicField;
    @NonNull
    String internalField;
    public PublicFieldsAllInit() {
        this.publicField = "bar";
        this.internalField = "foo";
    }
}

File: gluecodium/src/test/resources/smoke/internal_fields/output/android-kotlin/com/example/smoke/PublicFieldsAllInit.kt
/*

 *
 */

@file:JvmName("PublicFieldsAllInitExtensions")

package com.example.smoke


class PublicFieldsAllInit {
    @JvmField var publicField: String
    @JvmField @JvmSynthetic internal var internalField: String



    constructor() {
        this.publicField = "bar"
        this.internalField = "foo"
    }





}

